pkg/compute/store/boltdb/store.go

```diff
@@ -0,0 +1,439 @@
+package boltdb
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/bacalhau-project/bacalhau/pkg/compute/store"
+	"github.com/rs/zerolog/log"
+	bolt "go.etcd.io/bbolt"
+)
+
+const (
+	newExecutionComment = "Execution created"
+
+	DefaultSliceRetrievalCapacity = 10
+
+	BucketExecutionsName = "executions"
+	BucketHistoryName    = "history"
+	BucketJobIndexName   = "execution-index"
```

Comment:
I suggest to prefix the bucket names with `executions` as the compute node may use the same bolt database in the future to store info other than executiosn:

```
	BucketExecutionsName = "executions"
	BucketHistoryName    = "executions_history"
	BucketJobIndexName   = "executions_jobid_index"
```

or something similar